apps/build_plant_metabolic_model/display.yaml

```diff
@@ -3,9 +3,9 @@
 #
 name     : Build Plant Metabolic Model
 subtitle : |
-    Build a metabolic model for a plant based on annotated sequences of transcripts.
+    Build a metabolic model for a plant based on annotated sequences of coding sequences.
```

Comment:
Well, that sounds like it means that all that is annotated is where the coding sequences are. Is that the case?